Essentials/src/com/earth2me/essentials/Mob.java

```diff
@@ -24,6 +24,7 @@
     HORSE("Horse", Enemies.FRIENDLY, EntityType.HORSE),
     PIG("Pig", Enemies.FRIENDLY, EntityType.PIG),
     PIGZOMB("PigZombie", Enemies.NEUTRAL, MobCompat.ZOMBIFIED_PIGLIN),
+    ZOMBIFIED_PIGLIN("Zombified Piglin", Enemies.NEUTRAL, MobCompat.ZOMBIFIED_PIGLIN),
```

Comment:
Name can't have a space in it, also this is basically a duplicate of PIGZOMB, cause they are both exactly the same mob (except for texture). I intentionally didn't add a new one because people can just continue using pigzombie.